_layouts/documentation.html

```diff
@@ -298,6 +298,10 @@
                         <summary class="nav-header">Tutorials</summary>
 
                         <ul class="nav nav-pills nav-stacked">
+                            <li
+                            {% if page.menu == 'check' %} class="nav--vertical__active" {% endif %}>
+                            <a href="/documentation/tutorials/check">Check</a></li>
```

Comment:
Could the displayed text be `Generating reports with Check`?

This would make it clearer what people would be getting out of the tutorial, which is arguably an issue with all of the other titles too but we can fix those another time